### ISSUE TITLE
(Interactive Description) Dynamic part of scene summary not fully resetting after pressing Reset All button

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#487]
Just noticed this while looking at the A11y View. 
![Screen Shot 2021-03-03 at 2 55 01 PM](https://user-images.githubusercontent.com/1902116/109853590-9da45880-7c30-11eb-83d9-d55928fb164e.png)

The description for the Sweater and Wall description after a reset should be:
- Sweater and wall have zero net charge, each has many pairs of negative and positive charges.

The description is correct on page load and on a full page refresh. For some reason it is not resetting on a Reset All.

**Comment by terracoda** [comment]
I'll update priority when/if I need this fixed for doing description course demos.

**Comment by terracoda** [comment]
Tagging @emily-phet and @jbphet so at next meeting we can discuss if this should be a blocking issue.

**Comment by jessegreenberg** [comment]
The problem is that when we re-generate the description after reset, the balloon has zero charges when the wall still has charge

![image](https://user-images.githubusercontent.com/6396244/126847909-d5ef11dc-628c-4a94-9bdb-4d7d6db17182.png)

This may be fine on its own, but I suspect that it is never corrected because the description doesn't correct itself after the sweater charges are reset.

**Comment by jessegreenberg** [comment]
This should be fixed in the above commit. Closing, but feel free to reopen if you would like to review @terracoda.
